perfkitbenchmarker/linux_benchmarks/cloudsuite_websearch_benchmark.py

```diff
@@ -12,197 +12,129 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Runs CloudSuite Web Search benchmark.
+"""Runs the Web Search benchmark of Cloudsuite.
 
-Docs:
-http://parsa.epfl.ch/cloudsuite/
-
-Runs CloudSuite Web Search to collect the statistics that show
-the operations completed per second and the minimum, maximum,
-average, 90th, and 99th response times.
+More info: http://cloudsuite.ch/websearch/
 """
 
-import posixpath
 import re
 import time
 
 from perfkitbenchmarker import configs
-from perfkitbenchmarker import vm_util
 from perfkitbenchmarker import flags
 from perfkitbenchmarker import sample
 
-from perfkitbenchmarker.linux_packages import solr as solr
-from perfkitbenchmarker.linux_packages import faban
-from perfkitbenchmarker.linux_packages.ant import ANT_HOME_DIR
-
 FLAGS = flags.FLAGS
 
-BENCHMARK_NAME = 'cloudsuite_websearch'
-BENCHMARK_CONFIG = """
-cloudsuite_websearch:
-  description: Run CloudSuite Web Search
-  vm_groups:
-    workers:
-      vm_spec: *default_single_core
-      disk_spec: *default_500_gb
-    clients:
-      vm_spec: *default_single_core
-"""
-
-CLOUDSUITE_WEB_SEARCH_DIR = posixpath.join(vm_util.VM_TMP_DIR,
-                                           'cloudsuite-web-search')
-
-FABAN_OUTPUT_DIR = posixpath.join(vm_util.VM_TMP_DIR, 'outputFaban')
-
-PACKAGES_URL = 'http://lsi-www.epfl.ch/parsa'
-INDEX_URL = posixpath.join(PACKAGES_URL, 'index')
-SOLR_CONFIG_URL = posixpath.join(PACKAGES_URL, 'solrconfig.xml')
-SCHEMA_URL = posixpath.join(PACKAGES_URL, 'schema.xml')
-SEARCH_DRIVER_URL = posixpath.join(PACKAGES_URL, 'search.tar.gz')
-
-SOLR_PORT = 8983
-
-CLASSPATH = ('$FABAN_HOME/lib/fabanagents.jar:$FABAN_HOME/lib/fabancommon.jar:'
-             '$FABAN_HOME/lib/fabandriver.jar:$JAVA_HOME/lib/tools.jar:'
-             '$FABAN_HOME/search/build/lib/search.jar')
-
-flags.DEFINE_string('cs_websearch_client_heap_size', '2g',
+flags.DEFINE_string('cs_websearch_client_heap_size',
```

Comment:
Oops, sorry, I see the flag was already defined in the original file. This might require more thought.